website/docs/guides/core-to-cloud-3.md

```diff
@@ -0,0 +1,196 @@
+
+import CoretoCloudTable from '/snippets/_core-to-cloud-guide-table.md';
+
+<CoretoCloudTable/>
+
+## What you'll learn
+You may have already started your move to dbt Cloud and are looking for tips to help you optimize your dbt Cloud experience. This guide includes tips and caveats for the following areas:
+
+- [Adapters and connections](https://docs.getdbt.com/guides/core-to-cloud-3?step=3) 
+- [Development tools](https://docs.getdbt.com/guides/core-to-cloud-3?step=4) 
+- [Orchestration](https://docs.getdbt.com/guides/core-to-cloud-3?step=5)
+- [dbt Mesh](https://docs.getdbt.com/guides/core-to-cloud-3?step=6)
+- [dbt Semantic Layer](https://docs.getdbt.com/guides/core-to-cloud-3?step=7)
+- [dbt Explorer](https://docs.getdbt.com/guides/core-to-cloud-3?step=8)
+- [What's next?](https://docs.getdbt.com/guides/core-to-cloud-3?step=9)
+
+## Adapters and connections
+
+In dbt Cloud, you can natively connect to your data platform and test its [connection](/docs/connect-adapters) with a click of a button. This is especially useful for users who are new to dbt Cloud or are looking to streamline their connection setup. Here are some tips and caveats to consider:
+
+### Tips
+- Manage [dbt versions](/docs/dbt-versions/upgrade-dbt-version-in-cloud) and ensure team synchronization with dbt Cloud's one-click feature, eliminating the hassle of manual updates and version discrepancies.
+- dbt Cloud supports a whole host of [cloud providers](/docs/cloud/connect-data-platform/about-connections), including Snowflake, Databricks, BigQuery, Fabric, and Redshift (to name a few).
+- Use [Extended Attributes](/docs/deploy/deploy-environments#extended-attributes) to set a flexible [profiles.yml](/docs/core/connect-data-platform/profiles.yml) snippet in your dbt Cloud Environment settings. It gives you more control over environments (both deployment and development) and extends how dbt Cloud connects to the data platform within a given environment.
+  - For example, if you have a field in your `profiles.yml` that you’d like to add to the dbt Cloud adapter user interface, you can use Extended Attributes to set it.
+
+### Caveats
+- Not all parameters are available for adapters.
+- A project can only use one warehouse type.
+
+## Development tools
+
+dbt Cloud empowers data practitioners to develop in the tool of their choice. It ships with a [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) (local) or [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) (browser-based) to build, test, run, and version control your dbt projects.
+
+Both development tools are tailored to suit different audiences and preferences within your team. To streamline your team’s workflow, it's important to know who will prefer the dbt Cloud IDE and who might lean towards the dbt Cloud CLI. This section aims to clarify these preferences.
+
```

Comment:
@jtcohen6 quick one - based on [this thread](https://dbt-labs.slack.com/archives/C012TTNN0JK/p1712760453549829), should we also add info about the gitignore file as described [here](https://docs.getdbt.com/docs/collaborate/git/version-control-basics#the-gitignore-file)

> Make sure you check the .gitignore file contains the necessary entries. dbt Core doesn't interact with git so dbt Cloud doesn't automatically add or verify entries in the .gitignore file. Additionally, if the repository already contains dbt code and doesn't require initialization, dbt Cloud won't add any missing entries to the .gitignore file.